tests/ecs/ambiguity_detection.rs

```diff
@@ -1,82 +0,0 @@
-//! A test to confirm that `bevy` doesn't regress its system ambiguities count when using [`DefaultPlugins`].
-//! This is run in CI.
-//!
-//! Note that because this test requires rendering, it isn't actually an integration test!
-//! Instead, it's secretly an example: you can run this test manually using `cargo run --example ambiguity_detection`.
-
-use bevy::{
-    ecs::schedule::{InternedScheduleLabel, LogLevel, ScheduleBuildSettings},
-    prelude::*,
-    utils::HashMap,
-};
-use bevy_render::{pipelined_rendering::RenderExtractApp, RenderApp};
-
-fn main() {
-    let mut app = App::new();
-    app.add_plugins(DefaultPlugins);
-
-    let sub_app = app.main_mut();
-    configure_ambiguity_detection(sub_app);
-    let sub_app = app.sub_app_mut(RenderApp);
-    configure_ambiguity_detection(sub_app);
-    let sub_app = app.sub_app_mut(RenderExtractApp);
-    configure_ambiguity_detection(sub_app);
-
-    app.finish();
-    app.cleanup();
-    app.update();
-
-    let main_app_ambiguities = count_ambiguities(app.main());
-    assert_eq!(
-        main_app_ambiguities.total(),
-        // This number *should* be zero.
-        // Over time, we are working to reduce the number: your PR should not increase it.
-        // If you decrease this by fixing an ambiguity, reduce the number to prevent regressions.
-        // See https://github.com/bevyengine/bevy/issues/7386 for progress.
-        44,
-        "Main app has unexpected ambiguities among the following schedules: \n{:#?}.",
-        main_app_ambiguities,
-    );
-
-    // RenderApp is not checked here, because it is not within the App at this point.
-    let render_extract_ambiguities = count_ambiguities(app.sub_app(RenderExtractApp));
-    assert_eq!(
-        render_extract_ambiguities.total(),
-        0,
-        "RenderExtract app has unexpected ambiguities among the following schedules: \n{:#?}",
-        render_extract_ambiguities,
-    );
```

Comment:
This is at 0 ; so we might as well just keep this, to be sure it doesn't regress ?

I'd argue to keep the rest commented out, to be slightly more straightforward to test, but only print the total and remove the assert?

Another idea would be to compare ambiguity count with master, and offer a non blocking warning if the count is different, but it's slightly more work.